packages/website/docs/_components_pages/ai/Button/Button.mdx

```diff
@@ -15,6 +16,8 @@ import IconOnlyButton from "../../../_samples/ai/Button/IconOnlyButton/IconOnlyB
 
 ## More Samples
 
-### Icon-only AI Button with Menu in one of the states
+## AI Button with split mode state
+<ButtonSplitMenu />
 
+### Icon-only AI Button with Menu in one of the states
```

Comment:
Please consider capitalizing:

Icon-Only AI Button With Menu in One of the States

Also, please consider capitalizing for row 19: AI Button With Split Mode State